### ISSUE TITLE
Python3 support

### ISSUE THREAD

**Issue by richleland** [django-cumulus/django-cumulus#120]
'nuff said.

**Comment by shredding** [comment]
So ... is there news to this topic?

**Comment by ferrix** [comment]
Depends on these:

https://requires.io/github/django-cumulus/django-cumulus/requirements/?branch=master

I am not using py3 myself anywhere so I have not tested. I guess the 1.9 pyrax does not support py3.

**Comment by rasca** [comment]
Pyrax is now deprecated. We should be changing that

**Comment by ferrix** [comment]
Agreed. The latest version is working for me for now and that stuff is in maintenance, so I don't have a huge personal stake right now.
